odoo_2_odoo_data_transfer/models/odoo_data_transfer_template_line_mixin.py

```diff
@@ -216,7 +225,7 @@ def _parse_one2many_value(self, value):
             res.append((0, 0, parsed_rec_data))
         return res
 
-    def _parse_value(self, value):
+    def _parse_value(self, value, ids_map=False):
```

Comment:
Could you try using a single return?

Reply:
Yes, but this is a improve out of the purpose of this PR, can you ignore it?